roles/wazo-tox/tasks/main.yaml

```diff
@@ -35,6 +35,11 @@
       WAZO_TEST_DOCKER_OVERRIDE_EXTRA: "{{ docker_sibling_results.file }}"
   when: docker_install_siblings
 
+- name: Set the INTEGRATION_TEST_TIMEOUT environment variable
+  set_fact:
+    integration_test_timeout:
+      INTEGRATION_TEST_TIMEOUT: '{{ integration_test_timeout | default(omit) }}'
+
```

Comment:
I'm not sure if I need a `when` here. If I do, I'm not sure what it should be.

Reply:
no you don't need it